apps/storybook-react/stories/SideSheet.stories.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import { SideSheet, SideSheetProps, Button } from '@equinor/eds-core-react'
+import { Story, Meta } from '@storybook/react'
+
+const Child = styled.div`
+  padding: 6px;
+  background-color: rgba(255, 146, 0, 0.15);
+  box-sizing: border-box;
+  border: 1px dashed #ff9200;
+  border-radius: 4px;
+`
+
+export default {
+  title: 'Components/SideSheet',
+  component: SideSheet,
+  argTypes: {
+    variant: {
+      defaultValue: 'medium',
+      control: {
+        type: 'select',
+        options: ['small', 'medium', 'large', 'xlarge'],
```

Comment:
Can you avoid duplicating this here? I _think_ @vnys  did something here for the, hmm can't remember exactly which component, table maybe?

Reply:
Yup, this code was pretty much useless, just removed it 😄